### PR TITLE
Update information to point to patch submission instructions on wiki

### PR DESCRIPTION
**Pull request by rodrigc**
https://twistedmatrix.com/trac/ticket/8352

@zaheerm was able to follow these instructions here: https://github.com/twisted/twisted/pull/63#issuecomment-220750674
and was able to do the right thing.

### CODE REVIEW

**Comment by rodrigc** [comment]
@jvanasco mentioned here: http://twistedmatrix.com/pipermail/twisted-python/2016-May/030388.html
that it is possible for GitHub to show a warning box with the contents of CONTRIBUTING when making a PR.
This is mentioned in the GitHub docs here: https://help.github.com/articles/setting-guidelines-for-repository-contributors/ .  It might be good to configure this in the GitHub twisted repository.  @hawkowl ?

**Comment by jvanasco** [comment]
I think github just hiccuped for a bit.  It's showing up now
screen shot 2016-05-24 at 3 23 16 pm

**Comment by adiroiban** [comment]
the problem is that CONTRIBUTING will only produce a warning with a link.

Maybe is best to have this information as the PR template see this info here https://github.com/blog/2111-issue-and-pull-request-templates

What do you think?

**Comment by rodrigc** [comment]
@adiroiban I agree with you, providing a PR template as mentioned in  https://github.com/blog/2111-issue-and-pull-request-templates will definitely be helpful to steer people in the best direction to _"do the right thing"_

Merging the changes in this PR to CONTRIBUTING should be done as well though. :)

**Comment by rodrigc** [comment]
@jvanasco thank you for submitting to Twisted and providing your feedback and experience.  It is helpful!

One thing I noticed in your screenshot is that you are doing your changes on **trunk** and submitting your pull request against **trunk**.  I encourage you to have a read of: https://twistedmatrix.com/trac/wiki/TwistedDevelopment#SubmittingaPatch
and do your work on a branch in your new repository, and not on trunk.
Then, when you are ready to do the pull request, you can do the pull request against your branch.

This doesn't just apply to Twisted, but in general when working with projects on GitHub,
it use usually good practice to create a new branch specifically for a pull request.

**Comment by jvanasco** [comment]
@rodrigc 100% agreed. 

tldr; I read the entire thread today and disagreed with Glyph's proposed workflow. Before I posted anything I wanted to do a run-through on the current github setup, so just clicked "fork", added "foo.txt" and started a PR (which I wasn't going to submit).  I noticed there was no contributors warning, so posted to the list.  Then I noticed there was a 'contributors', so I tried renaming that to `contributors.md` and realized it must have just been some backend issue at GitHub.  an hour later it was working as expected.

**Comment by rodrigc** [comment]
@adiroiban updated the information in CONTRIBUTING: https://github.com/twisted/twisted/commit/9aadce642a751c7c6870bcbd66123741bd6eeb8e
